### ISSUE TITLE
Missing data from trace

### ISSUE THREAD

**Issue by veraciousnottaken** [dbeck121/CPI-Helper-Chrome-Extension#111]
Having Trace enabled there's information missing. Here I marked blocks that should have green background with Trace data.
![image](https://github.com/dbeck121/CPI-Helper-Chrome-Extension/assets/2012946/210a52cc-b794-43c5-bb73-54c43eea5bcc)

As you can see the information is available from original Trace screen (little blue envelopes).
![image](https://github.com/dbeck121/CPI-Helper-Chrome-Extension/assets/2012946/7b143a57-3906-487b-912d-3ce53302ab6a)

Sorry for low resolution images. I'm using: Cloud Integration Version - 5.53.8

**Comment by dbeck121** [comment]
Hi, sorry for the late resonse!

I only load the last x (not sure about the number) steps and your IFlow seems pretty big. So I guess this is what happens here. The problem is that I need one api call for each step and that puts a lot of preassure on the system. But I will check if there are some api improvements.

BR
Dominic

**Comment by veraciousnottaken** [comment]
That's right, in the same flow, if number of steps is more reasonable everything works like a charm! Additional stress to the API is that flow contains loops.

Could that X be configurable somehow?

**Comment by dbeck121** [comment]
Not yet but @incpi made great contributions to mir6 configurable options in the UI popup... I will check the variable and ask him for this additional field :)

**Comment by dbeck121** [comment]
Hi, I increased the number from 300 steps to 1000 (one step on screen can have multiple internal). I will test how stable it works and maybe will be part of next release.

BR
Dominic
